diff -urN mysql-5.5.21.orig/include/my_global.h mysql-5.5.21/include/my_global.h
--- mysql-5.5.21.orig/include/my_global.h	2012-01-31 12:28:14.000000000 +0100
+++ mysql-5.5.21/include/my_global.h	2012-03-05 17:39:13.000000000 +0100
@@ -505,6 +505,10 @@
 #undef DBUG_OFF
 #endif
 
+#if !defined(DBUG_ON) && !defined(DBUG_OFF)
+# define DBUG_OFF
+#endif
+
 /* We might be forced to turn debug off, if not turned off already */
 #if (defined(FORCE_DBUG_OFF) || defined(_lint)) && !defined(DBUG_OFF)
 #  define DBUG_OFF
