From: Clint Byrum <clint@ubuntu.com>
Bug-Ubuntu: http://pad.lv/894146
Forwarded: not needed
Description: Skip on AMD64 builds
Last-Update: 2012-05-29
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- /dev/null
+++ b/mysql-test/include/debian-fail-lp894146.inc
@@ -0,0 +1,5 @@
+if (`select @@version_compile_machine in ('x86_64')`)
+{
+  skip Fails intermittently on AMD64 buildds http://pad.lv/894146;
+}
+
--- a/mysql-test/suite/rpl/t/rpl_heartbeat_basic.test
+++ b/mysql-test/suite/rpl/t/rpl_heartbeat_basic.test
@@ -9,6 +9,7 @@
 # * Various states of master and heartbeat
 # * Circular replication
 #############################################################
+--source include/debian-fail-lp894146.inc
 --source include/master-slave.inc
 #
 # The test runs long and does not have any specifics to 
